Cargo.toml

```diff
@@ -1,21 +1,24 @@
 [package]
 name = "poseidon252"
-version = "0.10.0"
+version = "0.11.0"
 authors = [
   "zer0 <matteo@dusk.network>",	"vlopes11 <victor@dusk.network>",	"CPerezz <carlos@dusk.network>", "Kristoffer Ström <kristoffer@dusk.network>"
 ]
 edition = "2018"
 
 [dependencies]
-kelvin = "0.19.0"
-nstack = "0.5"
+nstack = { git = "https://github.com/dusk-network/nstack", branch = "vlopes11/bump-microkelvin" }
```

Comment:
This won't be merged until the git dep becomes a tag. I guess is pending the merge of: https://github.com/dusk-network/nstack/pull/15

Reply:
Fixed